js/datagrid.ts

```diff
@@ -135,18 +135,19 @@ export class DataGridModel extends DOMWidgetModel {
     const schema = Private.createSchema(data);
 
     if (this.data_model) {
-      // Need to update existing ViewBasedJSONModel's dataset attribute
-      // before discarding.
       this.data_model.updateDataset({ data: data.data, schema: schema });
```

Comment:
Do we need the `.updateDataset()` call here?? It looks like we're always creating a new model on line 143.

Reply:
Yeah, the commend I deleted alludes to it - but it's a tricky issue of some state remaining between different models. We handled that here: 

https://github.com/bloomberg/ipydatagrid/pull/232